### PR TITLE
Update time display due to Portugal hour change

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

File: lib/bokken_web/views/view_utils.ex [hunk 1]
```diff
@@ -28,16 +28,21 @@ defmodule BokkenWeb.ViewUtils do
...
-      "23:59"
+      "00:59"
   """
   def display_time(%DateTime{} = datetime) do
-    hour = two_characters(datetime.hour)
+    hour =
+      case datetime.hour do
+        23 -> 0 |> two_characters()
+        x -> (x + 1) |> two_characters()
+      end
```
Comment (ruilopesm) [inline lib/bokken_web/views/view_utils.ex]: To be honest, this doesn't really solves the problem. When Portugal change back to UTC, we would have to manually revert these changes.

Comment (MarioRodrigues10) [reply lib/bokken_web/views/view_utils.ex]: It solves in short term, for now i think its the best solution we can implement

Comment (ruilopesm) [reply lib/bokken_web/views/view_utils.ex]: Check out my most recent commit 👀

**Review by ruioliveira02 — APPROVED** [review]
